profile.html

```diff
@@ -41,30 +41,30 @@ <h2 class="card-title">Duckie Duckerson</h2>
                 <p class="card-text mb-3">Welcome to my profile! I am a new and happy quacker!</p>
             </div>  
         </section>
-        <section class="content d-flex justify-content-between w-75">
-            <div class="write-message-wrapper me-2">
-                <form id="write-message" class="card mb-3 d-flex flex-column p-3">
-                    <div class="d-flex">
-                        <div class="col-md-1 me-2">
+        <section class="d-flex justify-content-center row p-0 col-12 col-md-9 gap-3">
```

Comment:
```suggestion
        <section class="justify-content-center row p-0 col-12 col-md-9 gap-3">
```

Row is d-flex by default